src/approaches/oracle_approach.py

```diff
@@ -35,7 +35,7 @@ def get_gt_nsrts(predicates: Set[Predicate],
     elif CFG.env == "cover_typed_options":
         nsrts = _get_cover_gt_nsrts(options_are_typed=True)
     elif CFG.env == "cover_multistep_options":
-        nsrts = _get_cover_gt_nsrts(options_are_typed=True,
+        nsrts = _get_cover_gt_nsrts(options_are_typed=True, more_params=True,
```

Comment:
can we make this variable name more descriptive? maybe `include_robot_in_holding`?

Reply:
Good point, done.